### PR TITLE
SpikeHD.swerve version 1.2.0

### PR DESCRIPTION
**Pull request by SpikeHD**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/150730)

### CODE REVIEW

**Comment by SpikeHD** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\SpikeHD.swerve__DefaultSource\swerve.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\swerve.exe returned exit code: -1073741515

(Automated response - build 877.)

**Comment by SpikeHD** [comment]
@stephengillie Is there any way to see more of the logs? I'd like to know why it's crashing, because that seems like a wack error code that doesn't come up anywhere when I Google it.

Also this is an application that runs indefinitely (ie until the user stops the process with Ctrl+C), in case that has anything to do with it. Verification that it works could be done with the `-h` or `-v` flags.

**Comment by stephengillie** [comment]
Hi @SpikeHD,

Yeah, it's a bit of a quest. 
- Under the Details tab, click on `WinGetSvc-Validation`
- Click `View more details on Azure Pipelines`
- Under `Related`, find `4 published; 1 consumed` (The numbers might vary.)
- Find `InstallationVerificationLogs` and goto the right edge of that line. Click the 3 vertical dots.
- Click Download artifacts
- Unzip, and the contents will be the run logs. 

I wrote scripts which do this, filter for "error", "exception" and "fail", then post the results with GItHub's API.

**Comment by SpikeHD** [comment]
@stephengillie Thanks! Unfortunately it was... not as helpful as I would have liked:
```
"ErrorStream": "",
"OutputStream": "",
```

As you can probably imagine, I've tested this on my own Windows 64-bit machine and it's never failed with empty logs and a exit code akin to my student loan debt, so I'm unsure where to proceed from here 😄.

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-04-24 09:58:23.278 [FAIL] Installer failed security check. Url: https://github.com/SpikeHD/swerve/releases/download/v1.1.1/swerve-x86_64-pc-windows-msvc.exe Result: 0x80004005

(Automated response - build 877.)

**Comment by stephengillie** [comment]
Hi @SpikeHD,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 877.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\SpikeHD.swerve__DefaultSource\swerve.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\swerve.exe returned exit code: -1073741515

(Automated response - build 877.)

File: manifests/s/SpikeHD/swerve/1.1.1/SpikeHD.swerve.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.5-1-19041-4291.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: SpikeHD.swerve
+PackageVersion: 1.1.1
+InstallerLocale: en-US
+InstallerType: portable
+InstallModes:
+- silent
```
Comment (stephengillie) [inline manifests/s/SpikeHD/swerve/1.1.1/SpikeHD.swerve.installer.yaml:9]:
```suggestion
- silent
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with: 
image

**Comment by SpikeHD** [comment]
Changed this to target 1.2.0, which should remove the need for VCRUNTIME140 to be installed.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\SpikeHD.swerve__DefaultSource\swerve.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\swerve.exe returned exit code: -1073741515

(Automated response - build 877.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-04-26 07:54:48.849 [FAIL] Installer failed security check. Url: https://github.com/SpikeHD/swerve/releases/download/v1.2.0/swerve-x86_64-pc-windows-msvc.exe Result: 0x80004005

(Automated response - build 877.)

**Comment by stephengillie** [comment]
Hi @SpikeHD,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 877.)

**Comment by SpikeHD** [comment]
Again?? What are those error codes :sob:

File: manifests/s/SpikeHD/swerve/1.2.0/SpikeHD.swerve.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.5-1-19041-4291.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: SpikeHD.swerve
+PackageVersion: 1.2.0
+InstallerLocale: en-US
+InstallerType: portable
+InstallModes:
+- silent
+Installers:
```
Comment (stephengillie) [inline manifests/s/SpikeHD/swerve/1.2.0/SpikeHD.swerve.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**